CHANGELOG.md

```diff
@@ -580,6 +580,7 @@ Released with 1.0.0-beta.37 code base.
 -  Add optional hex formatting parameter for getTransactionrReceipt (#5153)
 -  Fix transactionRoot -> transactionsRoot in BlockHeader (#5083)
 -  Fix Promise in Accounts.signTransaction() throwing errors that cannot be caught (#4724)
+-  Fixed types for getPastEvents 
```

Comment:
1.7.5 is already frozen under RC so make it under 1.7.6, and add label future release